mlflow/utils/proto_json_utils.py

```diff
@@ -83,8 +80,8 @@ def default(self, o):  # pylint: disable=E0202
 
 
 def _dataframe_from_json(
-    path_or_str, schema: Schema = None, pandas_orient: str = "split", precise_float=False
-) -> pd.DataFrame:
+    path_or_str, schema=None, pandas_orient: str = "split", precise_float=False
+):  # -> pd.DataFrame
```

Comment:
Nit: I think we can just drop the `pd.DataFrame` comment since this is an internal utility & the documentation / method name both indicate that a dataframe is returned.